scrapers/covid-19/govScrapers/getIsrael.js

```diff
@@ -21,30 +22,43 @@ const params = {
  */
 const parseData = (data) => {
 	const dateRegex = /\d+-\d+-\d+/g;
-	const patientsPerDay = data[2].data;
-	const deadPatientsPerDay = data[3].data;
-	const recoveredPerDay = data[4].data.splice(20, data[4].data.length - 1);
-	const testsPerDay = data[5].data.splice(30, data[5].data.length - 1);
+	const patientsPerDay = data[1].data;
+	const deadPatientsPerDay = data[2].data;
+	const recoveredPerDay = data[3].data.splice(20, data[3].data.length - 1);
+	const testsPerDay = data[4].data.splice(30, data[4].data.length - 1);
 	const timeline = patientsPerDay.map((elem, index) => ({
 		date: elem.date.match(dateRegex)[0],
 		newHospitalized: elem.new_hospitalized,
-		totalhospitalized: elem.Counthospitalized,
+		hospitalPatients: elem.Counthospitalized,
+		homePationts: elem.patients_home,
+		hotelPationts: elem.patients_hotel,
 		totalBeds: elem.total_beds,
 		StandardOccupancy: elem.StandardOccupancy,
 		newDeaths: deadPatientsPerDay[index].amount,
 		newlyRecovered: recoveredPerDay[index].amount,
 		newTestsTaken: testsPerDay[index].amount,
-		newPositiveTests: testsPerDay[index].positiveAmount
+		newPositiveTests: testsPerDay[index].positiveAmount,
+		activeEasyStatus: elem.CountEasyStatus,
+		activeMediumStatus: elem.CountMediumStatus,
+		activeCriticalStatus: elem.CountHardStatus,
+		onVentilators: elem.CountBreath
 	}));
 	return {
 		updated: new Date(data[0].data.lastUpdate).valueOf(),
 		data: {
-			sickLocated: { home: data[1].data[0].amount, hospital: data[1].data[1].amount },
-			sickByAge: data[6].data.map((entry) => {
+			sickByAge: data[5].data.map((entry) => {
 				const { section, male, female } = entry;
 				return { section: `ages ${section} - ${section + 10}`, male, female };
 			}),
-			cityData: data[7].data.map((entry) => delete entry.status && entry),
+			missingStaff: {
+				verifiedDoctors: data[6].data[0].Verified_Doctors,
+				verifiedNurses: data[6].data[0].Verified_Nurses,
+				isolatedDoctors: data[6].data[0].isolated_Doctors,
+				isolatedNurses: data[6].data[0].isolated_Nurses,
+				isolatedOtherSector: data[6].data[0].isolated_Other_Sector
+			},
+			hospitalData: data[8].data,
+			cityData: data[7].data,
```

Comment:
should be `cityData: data[7].data.map((entry) => delete entry.status && entry)`

Reply:
Most of them are blank now your right, That's because most cities in Israel are not on lock down. But status value will be equal to "בסגר חלקי" (Partial lockdown) or "בסגר" (lockdown) when ever they are, And that is very useful information!
Please have a look at the data from the gov API, check out the fourth city in that array named "אשדוד" it has its status set to partial lockdown right now.

Reply:
There are a few that are on partial lock down right now. 
![Annotation 2020-07-06 102856](https://user-images.githubusercontent.com/12494197/86567938-caef1880-bf74-11ea-9568-2602b9d5ca7f.png)